tests/runtime/cpp/param-ownership/runner-coarse-borrowing.cpp

```diff
@@ -0,0 +1,43 @@
+//@ args = '--ownership coarse-borrowing'
+
+#include "runner_cpp.h"
+#include <array>
+int main() {
+  std::array<std::string_view, 2> a1 = {"value1", "value2"};
+  std::array<std::string_view, 2> a2 = {"value3", "value4"};
+  std::array<std::span<std::string_view const>, 2> as = {
+      std::span<std::string_view const>(a1.data(), a1.size()),
+      std::span<std::string_view const>(a2.data(), a2.size())};
+  std::span<std::span<std::string_view const> const> input(as.data(),
+                                                           as.size());
+  auto res = test::ownership::Foo(input);
+  assert(res.size() == 2);
+  assert(res[0].size() == 2);
+  assert(res[0][0].get_view() == "VALUE1");
```

Comment:
I guess this hand inlining is more readable than my previous equal overloading. I like it.